packages/main/package.json

```diff
@@ -66,6 +66,7 @@
     "copy:qunit": "cpx \"test/**/qunit/*.*\" dist/test-resources",
     "copy:webcomponents-polyfill": "cpx \"../../node_modules/@webcomponents/webcomponentsjs/**/*.*\" dist/webcomponentsjs/",
     "dev": "npm-run-all --sequential build:i18n watch:all",
+    "generate-new-component" : "node ./lib/create-new-component/create-new-component.js",
```

Comment:
Name could be "index.js" since you already have "create-new-component" in the directory